pkg/reconciler/v1alpha1/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -1031,11 +1031,11 @@ func TestValidateFrom_Invalid(t *testing.T) {
 }
 
 func TestResolvePipelineRun_withExistingTaskRuns(t *testing.T) {
-	names.TestingSeed()
-
+	// Not using names.TriggerSeed() here to ensure we would get a different taskrun name due to truncation if the base
+	// didn't match.
```

Comment:
without `TestingSeed` I thought the generated name would vary and im not sure why it seems to be always `pipelinerun-mytask-with-a-really-long-name-to-trigger-tru-9l9zj` in this test?

Reply:
Because the `TaskRun` already exists. =)

Reply:
(basically the test wouldn't fail without the fix with the `TestingSeed()` call)